### ISSUE TITLE
Support for Zoraxy as reverse proxy

### ISSUE THREAD

**Issue by essys** [scribble-rs/scribble.rs#367]
I am trying to host this behind Zoraxy ([https://github.com/tobychui/zoraxy](https://github.com/tobychui/zoraxy)).
WebSockets is supported but I assume manual config adjustments to the headers must be made.

I checked the wiki pages on reverse proxies:
- https://github.com/scribble-rs/scribble.rs/wiki/Reverse-proxy-(Apache)
- https://github.com/scribble-rs/scribble.rs/wiki/reverse-proxy-(nginx)

But I am not sure how to make this possible in Zoraxy.
Find attached the settings section for custom headers and the link to their wiki:
[https://github.com/tobychui/zoraxy/wiki/Custom-Headers](https://github.com/tobychui/zoraxy/wiki/Custom-Headers)

<img width="492" alt="Screenshot 2024-12-06 at 16 16 46" src="https://github.com/user-attachments/assets/7873153a-69f8-4dba-8e09-85ad88aaf217">

**Comment by Bios-Marcel** [comment]
Hm, not sure. Never heard of this tool.

The zoraxy readme claims that websockets are supported and should work out of the box.

So I guess the other headers might be missing. Did you see any concrete errors in the scribblers logs?

**Comment by essys** [comment]
FYI: Zoraxy is a new player in the FOSS reverse proxy market alongside other well known solutions like NGINX, Caddy or NPM.
It's been well maintained and so far I can only share very positive feedback about it.

> So I guess the other headers might be missing. Did you see any concrete errors in the scribblers logs?
Sorry, I missed this part completely.
I tried to fiddle around with the settings but I always got a '**404 page not found**'.

As I type this reply, I am embarrassed to admit that I have already found my mistake.
I was forwarding to the wrong port, I had to select the system port and not the one I had configured.
